app/views/submissions/show.html.erb

```diff
@@ -0,0 +1,11 @@
+<div>
+  <h2><%=@submission.title%></h2>
+  <h3><%=@submission.submitted_to%></h3>
+  <h3>Poems in this submission</h3>
+  <ul>
+    <% if @submission&.poems&.each do |poem| %>
```

Comment:
> I mistakenly thought that calling each on an empty array produced an error too.

To really drive the point home, I recommend the 1985 classic [How to Replace Failure by a List of Successes](https://rkrishnan.org/files/wadler-1985.pdf), by Wadler.